addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -402,9 +392,30 @@ def _create_batch(self, user_id, picking_type_id, picking_priorities=None):
 
         return batch
 
+    def add_next_picking_to_batch(self, picking_type_id, picking_priorities):
+        """ Get the next available picking and add it to the current users batch """
+        Picking = self.env['stock.picking']
```

Comment:
Going to check with Product team, but this and remove_pickings() might be only needed for ephemeral batches.